src/client/ProposalClient.ts

```diff
@@ -260,14 +259,21 @@ class ProposalClient implements Proposal {
   }
 
   canExecute(): boolean {
-    if (this.zDAO.isRelativeMajority) return false;
+    if (!this.zDAO.isRelativeMajority) return false;
 
     const totalScore = this.scores.reduce((prev, current) => prev + current, 0);
-    const totalScoreAsBN = getDecimalAmount(
-      BigNumber.from(totalScore),
+    const totalScoreAsBN = parseUnits(
+      totalScore.toString(),
       this.zDAO.votingToken.decimals
     );
     if (totalScoreAsBN.gte(this.zDAO.minimumTotalVotingTokens)) {
+      // Assume score[0] is the score for Approval or Yes
+      // If Approval > Deny in score, can execute
+      for (const score of this.scores) {
+        if (score > this.scores[0]) {
```

Comment:
Not sure I understand this loop?

We iterate `this.scores`
Then, we check if the iteration is greater than `this.scores[0]`, which will be the same value of the first iteration, and X > X is always false, and so this loop always hits `return true` below?

Reply:
I just want to check if the first score is the largest score in `this.scores`.
So I can skip first score by removing `=` in `if` condition and if found any value is greater than first score, then returns false.
```javascript
const scores = [20, 15, 30, 19];
  for (const score of scores) {
    if (score > scores[0]) {
      return false;
    }
  }
```
In this example, 20 is less than 30, so should return false, but assume if first score is 30, then that is the largest value, returns true.
Please correct me if wrong.